packages/sdk/src/modules/did/index.ts

```diff
@@ -198,12 +327,23 @@ export class Did extends Base {
       this._getDidId(options.didControllerAddress.toString())
     );
 
-    const { verificationId, verificationMethod } =
-      this._createVerificationMethod(options.didAccountAddress.toString());
+    // do we want to preset a verification method? Or should we have user manually do it
+    if (options.customDocumentFields?.verifications) {
```

Comment:
Added loop for verifications, in case the user is trying to set multiple verification methods at the same time.